conans/requirements_dev.txt

```diff
@@ -1,6 +1,7 @@
 nose>=1.3.7, <1.4.0
 pytest>=6.1.1, <7.0.0; python_version > '3.0'
 pytest>=4.6.11; python_version < '3.0'
+pytest-xdist  # To launch in N cores with pytest -n
```

Comment:
I would wait before running tests in parallel in the CI, maybe it introduces some hard-to-find failure. Let's work first on the testing framework and then we can make this PR to introduce parallelism.

Reply:
Yes, I agree, this is not currently used in this PR, so can be omitted at the moment.